api/v1alpha1/dspipeline_types.go

```diff
@@ -131,6 +131,8 @@ type CABundle struct {
 	// the CA bundle file (e.g. ca-bundle.crt)
 	// +kubebuilder:validation:Required
 	ConfigMapKey string `json:"configMapKey"`
+	// +kubebuilder:validation:Required
```

Comment:
add description, mention that this is where the ca bundle will be mounted in pipeline server and user executor pods